lib/servers_regtest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "127.0.0.1": {
+        "pruning": "-",
+        "s": "50002",
+        "t": "50001",
```

Comment:
I guess these should rather be `'t': '51001', 's': '51002'`, which is also what you specified as `BitcoinRegtest.DEFAULT_PORTS`
https://github.com/kyuupichan/electrumx/blob/fbf0ec4d9b9bb5c4c6bdcbf27c3fd4d9def91670/lib/coins.py#L571

Reply:
Yep, changing now.